config/importmap.rb

```diff
@@ -1,5 +1,3 @@
-# Pin npm packages by running ./bin/importmap
```

Comment:
I think I'd prefer to leave this comment in the code. I'm not sure I'd ever remember how it works if we remove this.